src/lib/dialog/dialog.spec.ts

```diff
@@ -1045,6 +1052,97 @@ describe('MatDialog with a parent MatDialog', () => {
   }));
 });
 
+describe('MatDialog with default options', () => {
+  let dialog: MatDialog;
+  let overlayContainer: OverlayContainer;
+  let overlayContainerElement: HTMLElement;
+
+  let testViewContainerRef: ViewContainerRef;
+  let viewContainerFixture: ComponentFixture<ComponentWithChildViewContainer>;
+  let mockLocation: SpyLocation;
```

Comment:
You can also get rid of the variable and the place where you're injecting it below.

Reply:
Sorry, missed it